### DISCUSSION TITLE
Second chunked death sound

### DISCUSSION THREAD

**Discussion by lynxlynxlynx** [Spellhold-Studios/Infinity-Sounds#4] [Suggestions]
Congrats on the release! Two notes:
- iwds also shipped `gore2` for chunking a frozen foe (our code says also for petrified; sound fits both; I think this was RE from the forum). I guess it would require more tobex work though.

I know these games are not supported yet, but for future reference:
- original iwds have both sounds
- original pst is missing both sounds

**Comment by skellytz** [comment]
Thank you!
